k9mail/src/test/java/com/fsck/k9/mailstore/MessageViewInfoExtractorTest.java

```diff
@@ -48,8 +57,7 @@ public void testSimplePlainTextMessage() throws MessagingException {
         List<Part> outputNonViewableParts = new ArrayList<Part>();
         ArrayList<Viewable> outputViewableParts = new ArrayList<>();
         MessageExtractor.findViewablesAndAttachments(message, outputViewableParts, outputNonViewableParts);
-        ViewableExtractedText container = extractTextFromViewables(InstrumentationRegistry.getTargetContext(),
-                outputViewableParts);
+        ViewableExtractedText container = extractTextFromViewables(RuntimeEnvironment.application, outputViewableParts);
```

Comment:
Please don't use `RuntimeEnvironment.application` directly in tests. Assign it to a field `context` in `setUp()` and reference that instead.